contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -3662,7 +3661,7 @@ get_insert_bulk_kilobytes_per_batch()
 }
 
 static const char 
-*gen_schema_name_for_fulltext_index(char *schema_name)
+*gen_schema_name_for_fulltext_index(const char *schema_name)
```

Comment:
I'm not clear on why this function exists separately from other fully qualified schema name resolutions - can you add a comment header to the function explaining why we need this? Is full text search the only case when this kind of fully qualified name is needed?

Reply:
Actually, it's just a helper function that I created specifically for getting the schema name for FTS. It's basically just a combination of two functions to handle the master db schema and custom schema case.